modules/data.land/inst/StatsModel_FIA.R

```diff
@@ -0,0 +1,88 @@
+#This code reads in output of CanopyCover2NLCD.r and using the thresholds of NLCD legens to convert them to NLCD classes. 
+library(rjags)
+
+#Reading in files
+PSS <- read.csv(file="/home/apourmok/My scripts/Sites/sitefiles.radius 0.075.lat 29.8848 lon -83.3542.pss.csv", head=TRUE, sep=",")
```

Comment:
I know this is just a demo script but please try to keep hard links out of the code base, if you put the csv in this directory you can retrieve it with `system.file` otherwise you can store it in betydb and retrieve its location